frontend/src/global_styles/common/header/logo.sass

```diff
@@ -4,12 +4,12 @@
   margin: 0 0.5rem
 
   &--link
-    margin-top: 16px
+    margin: 0.25rem 0
```

Comment:
I guess it makes sense to extract that into a local variable so that you can re-use it when calculating the height.